routes/controllers/manageUsers.js

```diff
@@ -0,0 +1,190 @@
+const {
+  registerUser,
+  loginUser,
+  currentUser,
+  updateSubscription,
+} = require("../../models/users");
+const { User } = require("../../service/schema/user.schema");
+
+const signupUser = async (req, res, next) => {
+  try {
+    const body = {
+      email: req.body.email,
+      password: req.body.password,
+    };
+    console.log(req.body);
+    const result = await registerUser(body);
+    if (result && result !== 409 && result.status !== 400) {
+      await result.save();
+      return res.status(201).json({
+        status: "success",
+        code: 201,
+        data: {
+          user: {
+            email: result.email,
+            subscription: result.subscription,
+          },
+        },
+      });
+    }
+    const message = result.message || "Email in use";
+    return res.status(result.status || 409).json({
+      status: "error",
+      code: result.status || 409,
+      message,
+    });
+  } catch (err) {
+    console.log(err.message);
```

Comment:
```suggestion
    console.log(err);
```
Tu lepiej cały błąd bo masz więcej informacji gdzie problem.